README.md

```diff
@@ -58,8 +58,13 @@ class UserModel {
 
 }
 
+//posible options, for instance if you want to ignore getter and setters put
+options={
+	ignore_getters_and_setters:true
```

Comment:
Please name options in camelCase, also use let or const for declaring options here.